template/faas-flow/go.mod

```diff
@@ -1,23 +1,17 @@
 module handler
 
-go 1.14
+go 1.13
```

Comment:
Any specific reason for downgrading this?

Reply:
I think its a mistake

Reply:
Fixed